### PR TITLE
fix(dapp-svelte-wallet): add Far/Data annotations

### PR DESCRIPTION
**Pull request by warner**
I'm not sure this catches everything, but it allows the API unit tests to pass (while the `marshal.js` asserts are enabled).

refs #2018

### CODE REVIEW

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js [hunk 1]
```diff
@@ -491,15 +491,11 @@ export function makeWallet({
...
+    const paymentKeywordRecord = Data(
+      Object.fromEntries(keywords.map((keyword, i) => [keyword, payments[i]])),
     );
 
-    const seat = E(zoe).offer(
-      inviteP,
-      harden(proposal),
-      harden(paymentKeywordRecord),
-    );
+    const seat = E(zoe).offer(inviteP, harden(proposal), paymentKeywordRecord);
```
Comment (michaelfig) [inline packages/dapp-svelte-wallet/api/src/lib-wallet.js:498]: Why not harden `paymentKeywordRecord` as it was before?

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js [hunk 2]
```diff
@@ -778,7 +776,7 @@ export function makeWallet({
         origin,
         approvalP,
         enable: false,
-        actions: {
+        actions: Far('actions', {
```
Comment (michaelfig) [inline packages/dapp-svelte-wallet/api/src/lib-wallet.js]:
```suggestion
        actions: Far('dapp.actions', {
```

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js [hunk 3]
```diff
@@ -255,7 +255,7 @@ export function makeWallet({
         ...jstate,
         purse,
         brand,
-        actions: {
+        actions: Far('actions', {
```
Comment (michaelfig) [inline packages/dapp-svelte-wallet/api/src/lib-wallet.js]:
```suggestion
        actions: Far('purse.actions', {
```

**Review by michaelfig — APPROVED** [review]
LGTM!  Thanks for doing this.

File: packages/dapp-svelte-wallet/api/src/lib-wallet.js (hunk 1 above)
Comment (warner) [reply packages/dapp-svelte-wallet/api/src/lib-wallet.js:498]: No strong reason, just that it seemed redundant with the `Data` applied a few lines earlier (which of course also does `harden`). Also it removes a tiny bit of `prettier`-imposed linecount bloat.

**Pull request merged**